cachi2/core/models/output.py

```diff
@@ -45,6 +47,12 @@ def from_package_dict(cls, package: dict[str, Any]) -> "Component":
         )
 
 
+class SbomMetadata(pydantic.BaseModel):
+    """Metadata field in a SBOM."""
+    timestamp: datetime = pydantic.Field(default_factory=datetime.utcnow)
```

Comment:
Pydantic states that `default_factory` is in [beta](https://docs.pydantic.dev/1.10/usage/models/#field-with-dynamic-default-value), but the alternative was to override the __init__ function, which felt worse than this. Thoughts?

Reply:
Btw, I added this `timestamp` field because I thought it was nice to have, but I can remove it altogether if you don't value in it.

Reply:
Nevermind, I got rid of the timestamp once I realized we'd need to change the integration tests to handle it properly :sweat_smile: